src/components/Filters/SchemaSieve.spec.js

```diff
@@ -995,6 +995,25 @@ describe('SchemaSieve', () => {
 
       expect(ajv.compile(filter)).not.toThrow()
     })
+
+    it.only('should work when "type" is an array', () => {
+      const schema = {
+        type: 'object',
+        properties: {
+          foo: { type: ['string', 'null'] },
```

Comment:
Does this mean that `foo` would be an array of strings or nulls? In which case would it have null values?

Reply:
@sradevski this means that `foo` can be of type `string` *or* `null`. A nice explanation can be found here https://json-schema.org/understanding-json-schema/reference/type.html#type